core/relay/config/Corda_Relay2.toml

```diff
@@ -1,40 +1,52 @@
-name = "Corda_Relay"
-port="9080"
+name = "Corda_Relay2"
+port="9082"
 host="localhost"
 hostname="localhost"
 db_path="db/Corda_Relay2/requests"
 # This will be replaced by the task queue.
 remote_db_path="db/Corda_Relay2/remote_request"
 
+# FOR TLS
+cert_path="credentials/fabric_cert.pem"
+key_path="credentials/fabric_key"
+tls=false
+
 [networks]
-[networks.Corda_Network]
+[networks.Corda_Network2]
 network="Corda"
 [networks.Dummy_Network]
 network="Dummy"
-[networks.americantfn]
-network="Corda"
 
 [relays]
 [relays.Fabric_Relay]
 hostname="localhost"
 port="9080"
-[relays.wtln_relay]
-hostname="tradelens.sl.cloud9.ibm.com"
-port="9089"
+tls=false
+tlsca_cert_path="credentials/fabric_ca_cert.pem"
+[relays.Fabric_Relay2]
+hostname="localhost"
+port="9083"
+tls=false
+tlsca_cert_path="credentials/fabric_ca_cert.pem"
 [relays.Dummy_relay]
 hostname="localhost"
 port="9085"
+tls=false
+tlsca_cert_path="credentials/fabric_ca_cert.pem"
 [relays.Corda_Relay]
 hostname="localhost"
 port="9081"
+tls=false
+tlsca_cert_path="credentials/fabric_ca_cert.pem"
 
 [drivers]
-[drivers.Fabric]
-hostname="localhost"
-port="9093"
 [drivers.Corda]
 hostname="localhost"
-port="9099"
+port="9098"
```

Comment:
Why change the port? If you do, keep in mind that the Getting Started docs show this port as 9099.

Reply:
I think are confused with this file, this file is for `Corda_Network2`'s relay, not `Corda_Network`'s relay.

Reply:
Got it.